packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -1613,3 +1627,15 @@ cp1Rusd_sdeusdParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address
 cp1Rusd_sdeusdParentConfig_priceHaircutUnscaled = "0.075"
 cp1Rusd_sdeusdParentConfig_autoExchangeDiscountUnscaled = "0.005"
 cp1Rusd_sdeusdParentConfig_oracleNodeId = "<%= settings.sdeusdUsdcNodeIdStork %>"
+# -----------------------------------------------
+# TODO later: pass depositingEnabled through var (but there's no parseBool)
+# cp1Rusd_rusd_depositingEnabled = "true"
+cp1Rusd_rselini_capUnscaled = "<%= formatEther(MaxUint256) %>"
+cp1Rusd_rselini_autoExchangeThresholdUnscaled = "0"
+cp1Rusd_rselini_autoExchangeInsuranceFeeUnscaled = "0"
+cp1Rusd_rselini_autoExchangeDustThresholdUnscaled = "0"
+cp1Rusd_rselini_bidSubmissionFeeUnscaled = "0"
+cp1Rusd_rseliniParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address %>"
+cp1Rusd_rseliniParentConfig_priceHaircutUnscaled = "0"
+cp1Rusd_rseliniParentConfig_autoExchangeDiscountUnscaled = "0"
+cp1Rusd_rseliniParentConfig_oracleNodeId = "<%= settings.rusdUsdNodeId %>"
```

Comment:
rusdUsdNodeId is essentially just a constant node? (to avoid creating a new constant node for this one as it's temporary)

Reply:
yes, this constant 1 node